src/yubikey.rs

```diff
@@ -609,15 +609,13 @@ impl YubiKey {
         touch_policy: u8,
     ) -> Result<(), Error> {
         let mut key_data = Zeroizing::new(vec![0u8; 1024]);
-        let templ = [0, Ins::ImportKey.code(), algorithm, key];
+        let templ = [0, Ins::ImportKey.code(), algorithm.into(), key.into()];
 
-        if key == YKPIV_KEY_CARDMGM
-            || key < YKPIV_KEY_RETIRED1
-            || (key > YKPIV_KEY_RETIRED20 && key < YKPIV_KEY_AUTHENTICATION)
-            || (key > YKPIV_KEY_CARDAUTH && key != YKPIV_KEY_ATTESTATION)
-        {
-            return Err(Error::KeyError);
-        }
+        // Only slot we want to exclude is CardManagement, which isn't in the enum.
+        // TODO: Decide whether to add it or not.
+        // match key {
+        //     SlotId::CardManagement => return Err(Error::KeyError),
+        // }
```

Comment:
I think we should exclude it, because I think it is disjoint from the key algorithms and instead is tied to 3DES.

Reply:
Yeah, go ahead and delete it

Reply:
Done.